packages/server/src/app/db/comments/comments.service.ts

```diff
@@ -34,6 +37,19 @@ export class CommentsService {
 
         let doc = await newComment.save();
         await this.blogsService.addComment(blogId);
+
+        const blogTitle = (await this.contentService.fetchOne(blogId, commentInfo.commentParentKind)).title;
+        const notification: CreateCommentNotification = {
+            kind: NotificationKind.CommentNotification,
+            sourceId: blogId,
```

Comment:
Originally, I had `sourceId` as the ID of the newly-created comment. I soon realized that wouldn't make sense, because there was no way anyone could be subscribed to a comment that _didn't yet exist_.

Fundamentally, `sourceId` should be "ID of the thing people are actually subscribed to".